test/testNestingDepth.py

```diff
@@ -0,0 +1,119 @@
+import unittest
+from .testHelpers import get_cpp_function_list
+
+
+class TestCppNestingDepth(unittest.TestCase):
+
+    def test_one_function_with_no_nd_condition(self):
+        result = get_cpp_function_list("int fun(){}")
+        self.assertEqual(0, result[0].max_nesting_depth)
+
+    def test_one_function_with_one_nd_condition(self):
+        result = get_cpp_function_list("int fun(){if(a){xx;}}")
+        self.assertEqual(1, result[0].max_nesting_depth)
+
+    def test_one_function_nd_with_question_mark(self):
+        result = get_cpp_function_list("int fun(){return (a)?b:c;}")
+        self.assertEqual(1, result[0].max_nesting_depth)
+
+    def test_one_function_nd_with_forever_loop(self):
```

Comment:
Question mark is interpreted as an if statement in [https://github.com/terryyin/lizard/pull/101/files#diff-7e6e46eb7ddafa74e71930de5e50f4ceR15](url)